Content and structure review for layouts tutorial (#248)
patternfly/training-scenarios: html-css/layouts/step5-instructions.md

> The gallery layout is designed so that all of its children are of uniform size, displayed horizontally and wrap responsively.

Reviewer: ```suggestion
The gallery layout makes sure all of its children are of uniform size, displayed horizontally and wrapped responsively.
```

&lt;/div&gt;
</pre>

2) <strong>Locate the parent of all six card components.</strong>

It will be the outermost `div`.

3) <strong>Add the gallery layout to the parent container located in step 2.</strong>

> The `gallery` will structure any children inside of the gallery into a grid layout, whereby the columns don’t extend beyond 250px in width, and when this happens they add more columns and wrap.

Reviewer: ```suggestion
The `gallery` will structure any children inside of the gallery into a grid layout, whereby the columns don’t extend beyond 250px in width, and when this happens, they add more columns and wrap.
```
